### ISSUE TITLE
Bidirectional Subroute color brilliance setting

### ISSUE THREAD

**Issue by Torqueitup** [KillBait/AutoDrive_Course_Editor#52]
I have difficulty in being able to clearly see the brown-colored bidirectional subroutes on the map when viewed from a zoomed out setting.  I have to zoom in quite a way to be able to see the individual junctions.  The contrast between the brown routes and the background field settings is not so good.
Is there any workaround to this issue?

**Comment by KillBait** [comment]
Not currently, but i do have something coming that may help ( it is very W.I.P at the moment )

![Editor_Alpha](https://user-images.githubusercontent.com/7914589/187460718-102f61ce-e14e-4a54-887a-b6fe15b39e8b.png)

**Comment by Torqueitup** [comment]
Yup, the color wheel should work.  My solution which worked OK was to load the png into a graphics program - GIMP, Irfanview, ACDSee etc and change the brightness/contrast of the color using the eyedropper to select the correct color.  Then I saved the image and used that.  It worked well.  But if something can be built into the overall editor, then that would be even better.
Keep up the good work.

**Comment by KillBait** [comment]
Hopefully the latest version has fixed the issue you were having, feel free to let me know if there is anything
else you can think of that will help.

**Comment by Torqueitup** [comment]
The latest version is a real show-stopper and is totally brilliant.  Gee it makes editing so much better now.  Thanks heaps for your work.
